apps/frontend/src/pages/quiz/libs/components/analizing/analizing.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	onNext: () => void;
+};
+
+const Analizing: React.FC<Properties> = ({ onNext }: Properties) => {
```

Comment:
```suggestion
const Analyzing: React.FC<Properties> = ({ onNext }: Properties) => {
```
You can use spell checker extension in VSCode to avoid such mistakes in future